### PR TITLE
Add modelmesh unit/fvt/mirror image  jobs and remove e2e test.

### PR DESCRIPTION
**Pull request by Jooho**
Add modelmesh serving unit test into CI.

### CODE REVIEW

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-images pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test  pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-images pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit-test pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse  pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-unit

**Comment by Jooho** [comment]
@anishasthana Finally, all tests passed including fvt(e2e). please review this.

File: ci-operator/config/opendatahub-io/modelmesh-serving/opendatahub-io-modelmesh-serving-main.yaml [hunk 1]
```diff
@@ -15,7 +32,7 @@ releases:
     release:
       architecture: amd64
       channel: stable
-      version: "4.10"
+      version: "4.11"
```
Comment (anishasthana) [inline ci-operator/config/opendatahub-io/modelmesh-serving/opendatahub-io-modelmesh-serving-main.yaml:35]: Can/should we just be using OpenShift 4.12?

Comment (Jooho) [reply ci-operator/config/opendatahub-io/modelmesh-serving/opendatahub-io-modelmesh-serving-main.yaml]: yes, agree. I change it to 4.12

**Comment by Jooho** [comment]
/pj-rehearse ci/rehearse/opendatahub-io/modelmesh-serving/main/modelmesh-serving-fvt

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-modelmesh-serving-main-modelmesh-serving-fvt

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**